cairo/plan.sh

```diff
@@ -1,30 +1,43 @@
 pkg_name=cairo
 pkg_origin=core
-pkg_version="1.14.8"
+pkg_version="1.14.10"
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=(
-  'LGPL-2.1'
-  'MPL-1.1'
+  "LGPL-2.1"
+  "MPL-1.1"
 )
 pkg_source="https://www.cairographics.org/releases/${pkg_name}-${pkg_version}.tar.xz"
-pkg_shasum="d1f2d98ae9a4111564f6de4e013d639cf77155baf2556582295a0f00a9bc5e20"
+pkg_shasum="7e87878658f2c9951a14fc64114d4958c0e65ac47530b8ac3078b2ce41b66a09"
 pkg_description="Cairo is a 2D graphics library with support for multiple output devices."
 pkg_upstream_url="https://www.cairographics.org"
 pkg_deps=(
+  core/bzip2
   core/expat
   core/fontconfig
   core/freetype
+  core/gcc-libs
   core/glib
+  core/glibc
+  core/libffi
+  core/libiconv
   core/libpng
+  core/libxau
+  core/libxcb
+  core/libxdmcp
+  core/libxext
   core/pcre
   core/pixman
+  core/xlib
   core/zlib
 )
 pkg_build_deps=(
-  core/diffutils
+  core/autoconf
```

Comment:
Why do we need autoconf and automake added here?

Reply:
Without it the build wouldn't pass. I didn't exactly looked into why. Might have time to have a look tomorrow ;)

Reply:
You need to keep 'diffutils', too. The _build_ will succeed, but the configure step will not produce optimal Makefiles.